### PR TITLE
Hello, AppEngine!

### PR DESCRIPTION
**Pull request by zyme**
Relatively small set of changes here--a few fixes to some imports for case-sensitivity in docker,  a few Dockerfiles  to manage the builds, and some shell script glue to pull together builds and deploys.

### CODE REVIEW

File: Dockerfile-code [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM study-manager-base:latest
```
Comment (zyme) [inline Dockerfile-code:1]: Multistage builds in docker are pretty great--let one Dockerfile manage stuff that rarely changes, and then depend on it as infrastructure for a different Dockerfile.

File: Dockerfile-base [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FROM nginx:1.14.2
```
Comment (zyme) [inline Dockerfile-base:1]: I went with "least disruptive", and stuck with `nginx` as the base, even though we don't need nginx.  These dockerfiles are _just_ for building the angular stuff and writing out to `dist`.

File: src/app/medical-record/medical-record.component.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ import {PatchUtil} from "../utils/patch.model";
 import {MedicalRecord} from "./medical-record.model";
 import {PDFModel} from "../pdf-download/pdf-download.model";
 
-var fileSaver = require( "file-saver/filesaver.js" );
+var fileSaver = require( "file-saver/FileSaver.js" );
```
Comment (zyme) [inline src/app/medical-record/medical-record.component.ts:23]: Case-sensitivity in unix was causing docker-based builds to fail.

**Review by MocanaAtBroad — APPROVED** [review]

**Review by MocanaAtBroad — COMMENTED** [review]
No CircleCi?

File: src/assets/js/ddp_config.js.ctmpl [hunk 4]
```diff
@@ -1,20 +0,0 @@
...
-  'production': {{$ui.production}},
-  'environment': '{{$environment}}'
-};
-
-
-{{end}}
-{{end}}
-{{end}}
-{{end}}
-{{end}}
```
Comment (MocanaAtBroad) [inline src/assets/js/ddp_config.js.ctmpl:20]: We have an alternate way to produce this file or obtain this configuration?

**Pull request merged**